cmd/buildah/inspect.go

```diff
@@ -96,7 +97,11 @@ func inspectCmd(c *cli.Context) error {
 	}
 
 	if c.IsSet("format") {
-		return t.Execute(os.Stdout, buildah.GetBuildInfo(builder))
+		if err := t.Execute(os.Stdout, buildah.GetBuildInfo(builder)); err != nil {
+			return err
+		}
+		fmt.Println()
```

Comment:
Do we want this if this is not a tty?  IE If I wanted to pype the output out, could this cause issues?

Reply:
Just tested it with pipe and writing to a file with ">" and its working fine.

Reply:
Well it works with a carriage return.

If I do a 
if [ "foobar" == `buildah inspect --format ''{{ .ID }}" ]; then 
...
fi
 
I would bet it would fail because buildah inspect --format will "return foobar\n"

Reply:
This is how its being done in podman as well for templates. Not sure how else to return a new line.

Reply:
We could do a terminal check before printing the extra line. since this is just for human beings

Reply:
Added the terminal check to print the extra line for us human beings XD